src/main/java/org/aion/wallet/connector/api/ApiBlockchainConnector.java

```diff
@@ -340,11 +341,12 @@ private TransactionDTO mapTransaction(final Transaction transaction) {
                 transaction.getNrgConsumed(),
                 transaction.getNrgPrice(),
                 transaction.getTimeStamp(),
-                TxState.FINISHED
+                TxState.FINISHED,
```

Comment:
We can drop this Enum if we're not gouing to use it. It will get confusing after a while